snpEff annotation null pointer exception

#### Reproduce
- Extract [test.zip](https://github.com/molgenis/molgenis/files/596812/test.zip) and import resulting file using the 'Upload' plugin with name 'testvcf'
- Select the 'testvcf' entity in the 'Data explorer' plugin
- Select the annotation tab
- Select the 'snpEff' annotator in the 'Available annotators' column (if the snpEff annotator is not available then configure it first)
- Select 'Annotate'

#### Expected
- Annotation successful

#### Observed
- ```java.lang.NullPointerException``` displayed above 1/1 progress bar

Server logs:
```
08:53:43.604 [molgenis-job-0] ERROR o.m.data.jobs.model.JobExecution - Failed
java.lang.RuntimeException: java.lang.NullPointerException
        at org.molgenis.data.annotation.web.CrudRepositoryAnnotator.annotate(CrudRepositoryAnnotator.java:106) ~[molgenis-data-annotators-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at org.molgenis.data.annotation.web.CrudRepositoryAnnotator.annotate(CrudRepositoryAnnotator.java:51) ~[molgenis-data-annotators-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at org.molgenis.dataexplorer.controller.AnnotationJob.call(AnnotationJob.java:51) ~[molgenis-dataexplorer-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at org.molgenis.dataexplorer.controller.AnnotationJob.call(AnnotationJob.java:18) ~[molgenis-dataexplorer-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at org.molgenis.data.jobs.Job.runWithAuthentication(Job.java:89) ~[molgenis-jobs-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at org.molgenis.data.jobs.Job.tryRunWithAuthentication(Job.java:74) ~[molgenis-jobs-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at org.molgenis.data.jobs.Job.lambda$call$0(Job.java:40) ~[molgenis-jobs-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:133) ~[spring-tx-4.1.8.RELEASE.jar:4.1.8.RELEASE]
        at org.molgenis.data.jobs.Job.call(Job.java:40) ~[molgenis-jobs-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_112]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_112]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_112]
        at java.lang.Thread.run(Thread.java:745) [na:1.8.0_112]
Caused by: java.lang.NullPointerException: null
        at org.molgenis.data.validation.RepositoryValidationDecorator.lambda$validateEntityValueReadOnly$15(RepositoryValidationDecorator.java:527) ~[molgenis-data-validation-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
        at java.util.ArrayList.forEach(ArrayList.java:1249) ~[na:1.8.0_112]
```

--- a/molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/web/CrudRepositoryAnnotator.java
+++ b/molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/web/CrudRepositoryAnnotator.java
@@ -75,9 +75,8 @@ public class CrudRepositoryAnnotator
 					Repository externalRepository = dataService.getMeta().createRepository(targetMetaData);
 					permissionSystemService.giveUserEntityPermissions(SecurityContextHolder.getContext(),
 							Collections.singletonList(externalRepository.getName()));
-					RunAsSystemProxy.runAsSystem(() -> dataService.getMeta().updateEntityType(AnnotatorUtils
-							.addAnnotatorMetaDataToRepositories(externalRepository.getEntityType(), attributeFactory,
-									annotator)));
+					RunAsSystemProxy.runAsSystem(
+							() -> dataService.getMeta().updateEntityType(externalRepository.getEntityType()));
 
 					iterateOverEntitiesAndAnnotate(repository, annotator, DatabaseAction.ADD);
 				}

